docs/UserGuide.md

```diff
@@ -1082,8 +1082,8 @@ Fig 11. Revenue bar chart
 Fig 12. Client pie chart
 {:.caption}
 
-#### 4.7.3 Viewing top 10 recipes
-* Your top 10 recipes that appear in the highest number of orders will be down in a pie chart.
+#### 4.7.3 Viewing top 10 most popular recipes
+* Your top 10 most popular recipes, that appear most frequently in the all your orders will be shown in a pie chart.
```

Comment:
Could you add a line that this is done by comparing total order quantity for each recipe? E.g. 1 order with Chicken Rice x3 is more frequent than another order with Chicken Rice x2